src/Components/InputTodo.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Please it would be best to solve any linter errors in your code rather than disabling linter checks. it helps you write better code.